src/deepsparse/transformers/pipelines/question_answering.py

```diff
@@ -160,6 +160,16 @@ def __init__(
 
         super().__init__(**kwargs)
 
+        if doc_stride is None:
```

Comment:
let's see if we can read the number of special tokens from the tokenizer or tokenizer config and cap doc stride at `min(doc_stride, sequence_len - num_special_tokens)`

Reply:
and then let's add short warning